### PR TITLE
[Composer]: Disable changing from field, added toottip to icons and enlarged attachment icon

### PR DESCRIPTION
**Pull request by pooja169usp**
# Code changes

- Replaced `<nylas-contact-search>` in the from field with html
- Enlarged attachment icon [sc-77498](https://app.shortcut.com/nylas/story/77498/composer-attachment-icon-too-small)
- Added `title` attribute to the icons (Bold, Italic etc) [sc-77496](https://app.shortcut.com/nylas/story/77496/composer-no-tooltip-labels-are-shown-when-hovered-over-the-cta-button-icons-present-in-the-composer)
- Added a test to verify that from field cannot be changed / removed
- Removed the test which checked to see that from field can be removed

# Screenhot
image

# Readiness checklist

- [X] Added changes to component `CHANGELOG.md`
- [X] Cypress tests passing?
- [x] New cypress tests added?
- [X] Included before/after screenshots, if the change is visual

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: components/composer/src/init.spec.js [hunk 1]
```diff
@@ -471,7 +495,7 @@ describe("Composer customizations", () => {
     });
   });
 
-  it.only("Replaces merge fields as defined in replace_fields when passed a prop", () => {
+  it("Replaces merge fields as defined in replace_fields when passed a prop", () => {
```
Comment (krisgardiner) [inline components/composer/src/init.spec.js:498]: Thanks 🙂

**Review by krisgardiner — APPROVED** [review]

File: components/composer/src/Composer.svelte [hunk 2]
```diff
@@ -677,14 +717,21 @@
...
+                  <div class="contact-item" data-cy="from-field">
+                    <span class="contact-item__name">
+                      {#if $message.from[0].name}
+                        <strong>{$message.from[0].name}</strong>
+                        {`<${$message.from[0].email}>`}
+                      {:else}{$message.from[0].email}{/if}
+                    </span>
+                  </div>
+                </div>
+              </div>
```
Comment (AaronDDM) [inline components/composer/src/Composer.svelte:733]: @hazik @pooja169usp so we're never supporting changing of the from field?

File: components/mailbox/src/index.html [hunk 3]
```diff
@@ -49,7 +49,7 @@
         const composer = document.querySelector("nylas-composer");
         composer.show_header = true;
         composer.show_minimize_button = false;
-        composer.show_from = false;
+        composer.show_from = true;
```
Comment (AaronDDM) [inline components/mailbox/src/index.html:52]: Do we need to make this change?

Comment (pooja169usp) [reply components/mailbox/src/index.html:52]: We don't really need it as it is only for the demo.

File: components/composer/src/Composer.svelte (hunk 2 above)
Comment (pooja169usp) [reply components/composer/src/Composer.svelte:733]: @AaronDDM we could add a prop that would enable/disable changing form field. 
But I wonder, when would someone want to change the from field? It is tied to the access token.

**Review by yifanplanet — APPROVED** [review]

**Pull request merged**